packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,100 @@
 
 .hds-side-nav__menu-toggle-button {
   position: absolute;
+  top: -2px;
+  left: calc(var(--hds-app-sidenav-border-width) * -1);
   z-index: 1;
+  display: flex;
+  box-sizing: content-box;
```

Comment:
maybe add a short comment here, explaining why we're using `content-box` for this element